.cirrus.yml

```diff
@@ -0,0 +1,55 @@
+BUILD_TEST_TASK_TEMPLATE: &BUILD_TEST_TASK_TEMPLATE
+  arch_check_script:
+    - uname -am
+  test_script:
+    - python --version
+    - python -m pip install --upgrade pip
+    - python -m pip install -r requirements_dev.txt
+    - python -m flake8
+    - python -m pydocstyle pact
+    - python -m tox -e test
+    # - make examples
+
+linux_arm64_task: 
+  env:
+    matrix:
+      # - IMAGE: python:3.6-slim  # This works locally, with cirrus run, but fails in CI
```

Comment:
Failed in CI

https://cirrus-ci.com/task/5786844193882112

<img width="731" alt="Screenshot 2023-04-21 at 16 07 08" src="https://user-images.githubusercontent.com/19932401/233670957-f3780b8d-5c85-4f94-b4c8-1dbbc8e26757.png">


<img width="948" alt="Screenshot 2023-04-21 at 16 06 49" src="https://user-images.githubusercontent.com/19932401/233670761-a2827f8f-ba0c-4972-952f-3a0866308ad8.png">